doc/nrf/libraries/bluetooth_services/mesh/vnd/le_pair_resp.rst

```diff
@@ -12,11 +12,22 @@ The LE Pairing Responder model can be used to hand over a passkey that will auth
 Before the pairing is initiated, an initiator should send an LE Pairing message with Passkey Reset sub-opcode to set a new passkey for the next pairing request.
 The passkey returned in the LE Pairing message with the Passkey Status sub-opcode should be used for the next pairing.
 
+The passkey is generated using the :c:func:`bt_rand` function and set to the host using the :c:func:`bt_passkey_set` function.
```

Comment:
Just realized that we don't mention anywhere that this is a vendor model. Can't add comments to the unchanged text. So adding it here.

> The LE Pairing Responder model **is a vendor model. This model** can be used to hand over a passkey that will authenticate a Bluetooth LE connection over a mesh network when it is not possible to use other pairing methods.